library/billing.inc

```diff
@@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
           "pid = '$patient_id' AND encounter = '$encounter_id'");
       }
     }
```

Comment:
yes they were. did not grep for that 8^(

      From: Brady Miller <notifications@github.com>
 To: openemr/openemr <openemr@noreply.github.com> 
Cc: Terry Hill <teryhill@yahoo.com>; Mention <mention@noreply.github.com>
 Sent: Thursday, August 18, 2016 6:36 PM
 Subject: Re: [openemr/openemr] Freeb removal (#187)
   
In library/billing.inc:> @@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
>            "pid = '$patient_id' AND encounter = '$encounter_id'");
>        }
>      }
@teryhill 
Just checking in with you. Weren't most of the above if (GLOBALS['oer_config']['ws_accounting']['enabled']) stuff removed by the sql-ledger removal?
(there are a couple more on a grep, and will then likely remove them with this PR)—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.